cms/services.ts

```diff
@@ -372,6 +372,10 @@ async function createMarkdownFromCSV(overwriteEntries: boolean = false) {
           })
         }
 
+        const image =
+          'img/fidfallback_' + Math.floor(Math.random() * 3 + 1) + '.jpg'
+        const imageAlt = 'Family Information Directory fallback image'
```

Comment:
A screenreader will say "image" as part of announcing the element, so we don't normally need it in the alt text too. I would probably set this alt text to "" as it is purely decorative and doesn't convey any info about the event.